types/office-js-preview/index.d.ts

```diff
@@ -58047,6 +59025,386 @@ declare namespace Excel {
+             * @beta
+             */
+            id?: boolean;
+            /**
+             *
+             * Represents the task's completion percentage. It is used for the "Progress" history record type.
+                        This is a value betwen 0 and 100, where 100 represents a completed task. Changing this value to 100 also completes the associated comment. Changing the completion from 100 to a lower value reactivates the associated comment.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            percentComplete?: boolean;
+            /**
+             *
+             * Represents the task's priority. It is used for the "Priority" history record type.
+                        This is a value between 0 and 10 with 5 being the default priority if not set, where 0 represents the highest priority.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            priority?: boolean;
+            /**
+             *
+             * Represents the task's start date, it is used for the "Schedule" history record type.
+                        It is in UTC time zone. Can be set to `null` to remove the start date. It should be set together with `dueDate` to avoid conflicts.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            startDate?: boolean;
+            /**
+             *
+             * Represents the task's title. It is used for "Title" history record type.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            title?: boolean;
+            /**
+             *
+             * Represents task history record's type.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            type?: boolean;
+            /**
+             *
+             * Represents the TaskHistoryRecord.id property that was undone for the "Undo" history record type.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            undoHistoryId?: boolean;
+        }
+        /**
+         *
+         * Represents a collection of history records for a task.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        interface TaskHistoryRecordCollectionLoadOptions {
+            /**
+              Specifying `$all` for the LoadOptions loads all the scalar properties (e.g.: `Range.address`) but not the navigational properties (e.g.: `Range.format.fill.color`).
+             */
+            $all?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Represents the ID of the object to which the task is anchored (e.g., commentId for tasks attached to comments).
```

Comment:
Reason for EACH ITEM to be capitalized (here and other places)?

Reply:
Not sure. There's a few LoadOptions phrasing issues @Rick-Kirkham has pointed out. We'll need to edit the part of codegen that produces all this task for every application-specific API. However, these aren't surfaced in the docs, so I'm okay leaving it as it is.